docs/cookbooks/fluentd/README.md

```diff
@@ -0,0 +1,149 @@
+  - [Step 3: Configure Custom Fluentd Plugins](#step-3-configure-custom-fluentd-plugins)
+  - [Step 4: Deploy Fluentd DeamonSet](#step-4-deploy-fluentd-deamonset)
+  - [Step 5: Visualize with Kibana Dashboard](#step-5-visualize-with-kibana-dashboard)
+- [Email Alerting](#email-alerting)
+<!-- /toc -->
+
+## Prerequisites
+
+The only prerequisites are:
+
+* a K8s cluster (Linux Nodes) running a K8s version supported by Antrea.
+* [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+All the required software will be deployed using YAML manifests, and the
+corresponding container images will be downloaded from public registries.
+
+## Practical steps
+
+### Step 1: Deploying Antrea
+
+For detailed information on the Antrea requirements and instructions on how to
+deploy Antrea, please refer to
+[getting-started.md](../../getting-started.md). To deploy the latest version of
+Antrea, use:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/antrea-io/antrea/main/build/yamls/antrea.yml
+```
+
+You may also choose a [released Antrea
+version](https://github.com/antrea-io/antrea/releases).
+
+### Step 2: Deploy Elasticsearch and Kibana Dashboard
+
+Fluentd supports multiple [output plugins](https://www.fluentd.org/plugins).
+Details will be discussed in [Step 4](#step-4-deploy-fluentd-deamonset), but
+by default, log records are collected by Fluentd DeamonSet and sent to Elasticsearch.
+A Kibana Dashboard can then be used to visualize the data. The YAML file for
+deployment is included in the `resources` directory. To deploy Elasticsearch
+and Kibana, run:
+
+```bash
+kubectl apply -f docs/cookbooks/fluentd/resources/kibana-elasticsearch.yml
+```
+
+### Step 3: Configure Custom Fluentd Plugins
+
+The architecture of Fluentd is a pipeline from input-> parser-> buffer->
+output-> formatter, many of these are plugins that could be configured to
+fit users’ different use cases.
+
+To specify custom input plugins and parsers, modify `./resources/kubernetes.conf`
+and create a ConfigMap with the following command. Later, direct Fluentd
+DaemonSet to refer to that ConfigMap. To see more variations of custom
+configuration, refer to
+[Fluentd inputs](https://docs.fluentbit.io/manual/pipeline/inputs).
+This cookbook uses the [tail](https://docs.fluentbit.io/manual/pipeline/inputs/tail)
+input plugin to monitor the audit logging files for Antrea-native policies
+on every K8s Node.
+
+```bash
+kubectl create configmap fluentd-conf --from-file=docs/cookbooks/fluentd/resources/kubernetes.conf --namespace=kube-logging
+```
+
+### Step 4: Deploy Fluentd DeamonSet
+
+Fluentd deployment includes RBAC and DeamonSet. Fluentd will collect logs
+from cluster components, so permissions need to be granted first through
+RBAC. Create a ServiceAccount, and grant permissions to read, list and
+watch Pods in ClusterRole. Then, use ClusterRoleBinding to link the two
+resources.
```

Comment:
the phrasing of this seems to imply that the user needs to create those resources manually, when my understanding is that they are defined in `fluentd.yml`

Maybe rephrase to:
> In `fluentd.yml`, we create a ServiceAccount, and use a ClusterRole and a ClusterRoleBinding to grant it permissions to read, list and watch Pods in cluster scope.

Then the last sentence is no longer necessary